### PR TITLE
Adds a delete url endpoint and tests that endpoint

### PR DESCRIPTION
**Pull request by qsingletary**
Description:

- Adds an endpoint for deleting a url
- Adds tests for tested that '/delete/ endpoint

### CODE REVIEW

File: server/src/routes/weburl.ts [hunk 1]
```diff
@@ -29,13 +29,24 @@ router.post('/create', async (req, res) => {
     }
 })
 
+router.delete('/delete', async (req, res) => {
```
Comment (sprakash57) [inline server/src/routes/weburl.ts:32]: I was thinking of having URL in the format `/delete/id`. So basically client will pass the id in the URL params and on the server side we need to delete the shorty on the basis of passed `id`. Passing `body` doesn't seem right to me.

File: server/src/__tests__/router.test.ts [hunk 2]
```diff
@@ -12,7 +12,17 @@ describe('Get endpoint', () => {
...
-describe('delete endpoint', () => {
+describe('Delete endpoint', () => {
+    beforeAll(async () => {
+        await request(server).post('/api/create').send({ inputUrl: 'some url' });
+    });
+    it('should delete a post input url', async () => {
+        const res = await request(server).delete('/api/delete').send({ inputUrl: 'some url' });
+        expect(res.body.message).toEqual('Url deleted');
+    })
+})
```
Comment (sprakash57) [inline server/src/__tests__/router.test.ts:23]: Awesome!! you have added test cases as well. Would you mind updating the README.md file as well, regarding the new API?

**Review by sprakash57 — CHANGES_REQUESTED** [review]
Thank you for your valuable contribution. Please see below review comments.

File: server/src/__tests__/router.test.ts (hunk 2 above)
Comment (qsingletary) [reply server/src/__tests__/router.test.ts:23]: Yup! I'll make the requested updates!

File: server/src/routes/weburl.ts (hunk 1 above)
Comment (qsingletary) [reply server/src/routes/weburl.ts]: @sprakash57 I have updated and pushed changes. Let me know if these updates work fine 👍

**Comment by sprakash57** [comment]
Great work. Thanks a lot.

**Pull request merged**